docs/content/getting-started/_index.md

```diff
@@ -44,12 +60,26 @@ There are a few options:
 3. Use [require.js](https://requirejs.org/) JavaScript file and module loader.
 	- Learn more about [Polyfills](/getting-started/polyfills)
 3. Load individual PatternFly Element scripts, but bundle the polyfills with the base `pfelement.js` file.
-	1.  All elements are based off of `pfelement.js` so including the polyfills with this one file would mean you only need to include the `pfelement.js` file before you include anything else.
+	-  All elements are based off of `pfelement.js` so including the polyfills with this one file would mean you only need to include the `pfelement.js` file before you include anything else.
 4. Bundle all of the scripts together into one rollup, and include that.
 
+### Important note
```

Comment:
@kylebuch8 idk if there's a better way to incorporate this info but I think it's vital to mention.

Reply:
I think this is fine.